src/api/projects.js

```diff
@@ -446,3 +448,73 @@ export const get = (event, context, callback) => {
       });
     });
 };
+
+/**
+ * Get all users not assigned to a specific project.
+ * @param  {object} event
+ * @param  {object} context
+ * @param  {object} callback
+ *  @return {function} done
+ */
+export const unassigned = (event, context, callback) => {
```

Comment:
Do we need to send project which are assigned to users?, From what I can see sending user's information is enough. We need to clarify this requirement

Reply:
@singhabahu I think it we need to ask from nilesh